boot/zephyr/nrf_cleanup.c

```diff
@@ -40,10 +48,28 @@ void nrf_cleanup_peripheral(void)
 #if defined(NRF_UARTE0)
     nrf_uarte_disable(NRF_UARTE0);
     nrf_uarte_int_disable(NRF_UARTE0, 0xFFFFFFFF);
+#if defined(NRF_DPPIC)
+    /* Clear all SUBSCRIBE configurations. */
+    memset((uint8_t *)NRF_UARTE0 + 0x80, 0, 0x80);
+    /* Clear all PUBLISH configurations. */
+    memset((uint8_t *)NRF_UARTE0 + 0x180, 0, 0x80);
+#endif
 #endif
 #if defined(NRF_UARTE1)
     nrf_uarte_disable(NRF_UARTE1);
```

Comment:
Any idea why mcuboot uses UARTE1?

Reply:
can be used for logging or serial recovery (depends on configuration)